content/docs/2.10/scalers/github-runner.md

```diff
@@ -0,0 +1,153 @@
++++
+title = "Github Runner Scaler"
+availability = "v2.10+"
+maintainer = "GitHub"
+description = "Scale GitHub Runners based on the number of queued jobs in GitHub Actions"
+go_file = "github_runner_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `github-runner-scaler` trigger that scales based on queued jobs in GitHub Actions.
+
+```yaml
+triggers:
+  - type: github-runner
+    metadata:
+      # Optional: The URL of the GitHub API, defaults to https://api.github.com
+      githubAPIURL: "https://api.github.com"
+      # Required: The owner of the GitHub repository, or the organization that owns the repository
+      owner: "{owner}"
+      # Required: The scope of the runner, can be either "org", "ent" or "repo"
+      runnerScope: "{runnerScope}"
+      # Required: The personal access token to access the GitHub API
+      personalAccessTokenFromEnv: "{personalAccessToken}"
+      # Optional: The list of repositories to scale, separated by comma
+      repos: "{repos}"
+      # Optional: The list of runner labels to scale on, separated by comma
+      labels: "{labels}"
+      # Optional: The target number of queued jobs to scale on
+      targetWorkflowQueueLength: "1" # Default 1
+      # Optional: The target number of queued jobs to achieve to activate the scaler
+      activationTargetWorkflowQueueLength: "0" # Default 0
+    authenticationRef:
+      name: personalAccessToken
+```
+
+**Parameter list:**
+
+- `githubAPIURL` - The URL of the GitHub API, defaults to https://api.github.com. You should only need to modify this if you have your own GitHub Appliance. (Optional)
+- `owner` - The owner of the GitHub repository, or the organization that owns the repository. (Required)
+- `runnerScope` - The scope of the runner, can be either "org", "ent" or "repo". (Required)
+- `personalAccessTokenFromEnv` - The personal access token to access the GitHub API. (Required)
```

Comment:
Is it really required if somebody uses TriggerAuthentication?

Reply:
removed meta element